essentials-framework/Essentials Devices Common/Essentials Devices Common/Cameras/CameraVisca.cs

```diff
@@ -171,7 +171,8 @@ public void LinkToApi(BasicTriList trilist, uint joinStart, string joinMapKey, E
 	    {
 	        LinkCameraToApi(this, trilist, joinStart, joinMapKey, bridge);
 
-			OnPresetsListHasChanged();
+			//OnPresetsListHasChanged();
```

Comment:
Lets maybe remove this line entirely instead of just leaving it commented out.  Don't want someone to uncomment it in the future

Reply:
@Lou-Iiacovelli See @ndorin's comment.